components/memberships/new-member-dialog.tsx

```
@@ -11,43 +16,67 @@ import {
+import { Label } from "../ui/label";
+import { PhoneInput } from "../ui/phone-number";
 import {
     Select,
     SelectContent,
     SelectItem,
     SelectTrigger,
     SelectValue,
 } from "../ui/select";
-import { useState } from "react";
+
 type NewMemberDialogProps = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)